Add terser as option to minify js (#153)
SAP/grunt-openui5: README.md

Optional parameter to set compression/minification of the files or to provide
additional options.

- JavaScript is minified using [terser](https://github.com/terser/terser)
- XML is minified using [pretty-data](https://github.com/vkiryukhin/pretty-data)
- JSON is parsed for correctness and to remove extra whitespace

An `object` can be used to provide options.  

> Provide `terser` object with the options. Object name `uflifyjs` is supported for backwards compatibility reasons to.

Reviewer: ```suggestion
Provide a `terser` object with the options. `uglifyjs` is also supported for backwards compatibility reasons.
```
